docs/data.rst

```diff
@@ -131,7 +131,7 @@ map creates a new stream where each element of the stream is the function
 applied to the corresponding element of the original stream. Evaluating the
 new stream will force evaluating the original stream up to that index.
 
-(Warning: This isn't the map builtin. In Python 3 the builtin map should do
+(Warning: This isn't the map built-in. In Python 3 the built-in map should do
```

Comment:
The styling used in the Python docs is "builtin": https://docs.python.org/3/library/builtins.html